specification/sql/resource-manager/readme.md

```diff
@@ -405,8 +449,7 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 output-folder: $(go-sdk-folder)/services/sql/mgmt/2014-04-01/sql
 ```
 
-
-## Python

```

Comment:
Please remove this Python section, there is currently two Python sections in this file and this one is the bad one.

Reply:
Done, thx